nevergrad/optimization/optimizerlib.py

```diff
@@ -762,6 +763,94 @@ def _internal_provide_recommendation(self) -> ArrayLike:
         return self.avg
 
 
+@registry.register
+class Splitter(base.Optimizer):
```

Comment:
I'd prefer to call this class SplitOptimizer since it isn't obvious that something called Splitter is an optimizer without looking at the source (I would expect a class called Splitter to have a split() method returning an optimizer or something else).

Reply:
Done.